### PR TITLE
#14:Feature/6 create team and price list

### PR DESCRIPTION
**Pull request by PhamNhuQuocBao**
image
image

### CODE REVIEW

**Comment by haunguyen15-tt** [comment]
Pls split into small PR

File: src/layouts/header.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<header>
+  <div class="container">
+    <div class="header_top">
+      <p class="header_top-logo">
+        hofmann
+      </p>
+      <nav class="header_top-navigation">
+        <ul>
+          <li class="item"><a href="">mission</a></li>
```
Comment (Lanng21) [inline src/layouts/header.html]: used loop

File: src/layouts/header.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <li class="item"><a href="">services</a></li>
```
Comment (Lanng21) [inline src/layouts/header.html]: and href ?

File: src/layouts/header.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          </form>
+          <div class="icon">
+            <div class="icon-plane">
+              <img src="../assets/icons/plane.svg" alt="Plane">
+            </div>
+            <div class="icon-wallet">
+              <img src="../assets/icons/wallet.svg" alt="Wallet">
+            </div>
+            <div class="icon-case">
+              <img src="../assets/icons/case.svg" alt="Case">
```
Comment (Lanng21) [inline src/layouts/header.html]: used figure tag

File: src/stylesheets/layouts/_header.css [hunk 4]
```diff
@@ -0,0 +1,272 @@
+@import url('../_reset.css');
+
+.header_top {
+  width: 100%;
+  height: 156px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: height

File: src/stylesheets/layouts/_header.css [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+
+.header_top-navigation {
+  display: flex;
+  align-items: center;
+}
+
+.header_top-navigation a {
+  text-transform: uppercase;
+  font-size: 18px;
+  color: #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: color

File: src/stylesheets/layouts/_header.css [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+ul .item {
+  margin: 0 22px;
+  text-align: center;
+}
+
+.item:last-child {
+  padding: 15px 25px;
+  border: 2px solid #ffffff;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+  border-radius: 50px;
+  width: 180px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: var(--text-color-primary);
+  opacity: 0.3;
+}
+
+.header_bottom-main {
+  height: 768px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

File: src/stylesheets/layouts/_header.css [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+form {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 60px;
+}
+
+.input-email {
+  width: 270px;
```
Comment (Lanng21) [inline src/stylesheets/layouts/_header.css]: same

**Comment by baonguyen26** [comment]
Fix conflict, pls

**Pull request merged**